app/views/provider_interface/application_choices/show.html.erb

```diff
@@ -73,15 +73,17 @@
     <%= render PersonalStatementComponent.new(application_form: @application_choice.application_form) %>
```

Comment:
This needs a `govuk-!-margin-bottom-2` like the others — maybe the heading should be pulled out of the component, as elsewhere?